package.json

```diff
@@ -160,7 +160,7 @@
     "@types/react": "16.9.5",
     "@types/react-router": "5.1.1",
     "react": "16.10.2",
-    "**/react": "16.10.1",
+    "**/react": "16.10.2",
```

Comment:
Otherwise we won't get the latest. Will try to remove resolutions in a follow up.